Add switch platform to honeywell (#32043)
home-assistant/home-assistant.github.io: source/_integrations/honeywell.markdown

- [Supported hardware](#supported-hardware)
- [Climate](#climate)
- [Sensor](#sensor)

> - [Switch](#switch)

Reviewer: ```suggestion
- [Switch](#switch)

{% include integrations/config_flow.md %}
```

Author: Thanks!

|Outdoor humidity | Average humidity of all Honeywell Wireless Outdoor Sensors|
|Indoor temperature | Current temperature as measured at the specific thermostat|
|Indoor humidity | Current humidity as measured at the specific thermostat|

> ## Switch

Reviewer: ```suggestion
## Switch

```

> The switch platform integrates the emergency heat option as a switch for each device.  If the thermostat supports emergency heat, the switch entity will be created.

Reviewer: ```suggestion
The switch entity integrates the emergency heat option for each device.  If the thermostat supports emergency heat, the switch entity will be created.
```

> This integration will add Home Assistant switch for the following:

Reviewer: ```suggestion
This integration will add a switch for the following:
```

> |Sensor|Value|

Reviewer: ```suggestion
|Switch|Value|
```
